### PR TITLE
PBC-2693: Update vertex integration guide.

### PR DESCRIPTION
**Pull request by danielsantos-spryker**
## PR Description
- Update Vertex integration guide.

Ticket: https://spryker.atlassian.net/browse/PBC-2693

TBD

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md [hunk 1]
```diff
@@ -9,6 +9,17 @@ related:
...
+
+Before you can integrate Vertex, make sure that your project is ACP-enabled. See [App Composition Platform installation](/docs/acp/user/app-composition-platform-installation.html) for details.
+
+The Vertex app requires the following Spryker modules:
+
+* `spryker/tax-app: ^0.1.0`
+* `spryker-shop/cart-page: ^3.38.0`
+* `spryker/product-offer-availability: ^1.4.0` (Marketplace only)
+* `spryker/merchant-profile: ^1.5.0` (Marketplace only)
+
```
Comment (brunoviana-spryker) [inline docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md:22]: I think we need also `"spryker/stock-address": "^1.2.0"`

Comment (danielsantos-spryker) [reply docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md:23]: added

**Review by matweew — COMMENTED** [review]
I don't think that `202307.0` is correct path of this page, because it's new functionality and will appear only in the next release.

So even having  the page already published here https://docs.spryker.com/docs/pbc/all/tax-management/202307.0/vertex/install-vertex.html  we need to move it to next version.

**Comment by danielsantos-spryker** [comment]
> I don't think that `202307.0` is correct path of this page, because it's new functionality and will appear only in the next release.
> 
> So even having the page already published here https://docs.spryker.com/docs/pbc/all/tax-management/202307.0/vertex/install-vertex.html we need to move it to next version.

@lenadoc what do you think about this?

File: docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md [hunk 2]
```diff
@@ -142,44 +226,13 @@ use Spryker\Zed\TaxApp\Communication\Plugin\Oms\Command\SendPaymentTaxInvoicePlu
...
-        return $pluginStack;
-    }
+## Integrate the Vertex app
 
-// ...
-```
+Spryker does not have the same data model as Vertex requires to do correct taxes calculation, so the integration requires from the project developers to add some missing information to the Quote object before sending calculation request.
 
-## Integrate the Vertex app
+ [image here]
```
Comment (matweew) [inline docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md:233]: image

Comment (matweew) [reply docs/pbc/all/tax-management/202307.0/vertex/install-vertex.md:233]: @lenadoc please add it this picture instead of `[image here]`

**Review by matweew — APPROVED** [review]

**Comment by lenadoc** [comment]
> > I don't think that `202307.0` is correct path of this page, because it's new functionality and will appear only in the next release.
> > So even having the page already published here https://docs.spryker.com/docs/pbc/all/tax-management/202307.0/vertex/install-vertex.html we need to move it to next version.
> 
> @lenadoc what do you think about this?

I totally agree. I've moved Vertex to the new version and added redirects from the current version to the new one for Vertex-related docs

**Pull request merged**